Sources/ssg/SwiftStringGarbler.swift

```diff
@@ -147,19 +150,20 @@ final class SwiftStringGarbler {
         return nil
     }
 
-}
-
-extension AbsolutePath {
-    func existsAsFile(in fileSystem: FileSystem = localFileSystem) -> Bool {
-        return fileSystem.isFile(self)
-    }
-}
-
-extension String {
-    @available(OSX 10.15, *)
-    func sha256Checksum() -> String {
-        guard let d = data(using: .utf8) else { fatalError("Can't get data representation of string \(self)") }
-        let digest = SHA256.hash(data: d)
-        return digest.compactMap { String(format: "%02x", $0) }.joined()
+    private func variablesExtractedFromWhereReport(variables: [ValueTuple]) -> String {
```

Comment:
Nitpick: I was kinda confused by this name at first glance. Thoughts about renaming it to something such as `extractedVariablesLocationReport` or `reportExtractedVariableLocations`?